command/command_test.go

```diff
@@ -52,6 +52,21 @@ func testCtxConfig(p terraform.ResourceProvider) *terraform.ContextOpts {
 	}
 }
 
+func testCtxConfigWithShell(p terraform.ResourceProvider, pr terraform.ResourceProvisioner) *terraform.ContextOpts {
```

Comment:
I duplicated the existing testCtxConfig function and added in the shell provisioner so I could determine from execution time whether or not the parallelism flag is functioning. Seems like I could DRY this up a bit, but this works for now.